src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -1324,13 +1324,13 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
     }
 
     /**
-     * DELETE /courses/:courseId/tutors/:tutorsLogin : Remove the given user from the tutors of the course so that the tutors cannot access the course administration any more
+     * DELETE courses/:courseId/tutors/:tutorsLogin : Remove the given user from the tutors of the course so that the tutors cannot access the course administration any more
```

Comment:
```suggestion
     * DELETE /courses/:courseId/tutors/:tutorsLogin : Remove the given user from the tutors of the course so that the tutors cannot access the course administration any more
```